src/js/controllers/index.js

```diff
@@ -624,6 +622,22 @@ angular.module('copayApp.controllers').controller('indexController', function($r
 
   $rootScope.$on('Local/BWSUpdated', function(event) {
     profileService.applyConfig();
+    storageService.setCleanAndScanAddresses(function() {});
+  });
+
+  $rootScope.$on('Local/CleanAndScanAddresses', function(event) {
+    storageService.getCleanAndScanAddresses(function(err, val) {
```

Comment:
I think this should not be an other event, but it should be in 'newfocusedwallet'